domains/sty.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Sty928",
+        "email": "satyam99jha@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
```

Comment:
Is-a-dev records, will not work here.

Reply:
I cannot get .is-cool.dev if i have a .is-a.dev domain?